sdk/servicebus/service-bus/test/stress/values.yaml

```diff
@@ -0,0 +1,10 @@
+scenarios:
+  - scenarioBatchReceive
```

Comment:
A comment in this file would useful, just to give some context on where this file gets consumed, etc..

Reply:
This says optional list of scenarios, but I'm not sure I understand why it's optional for service bus. This just ensures that helm will create 1 instance of the job for each scenario, right? (Is it just optional because someone using this as an example may not need it?)

Reply:
Correct, you don't need to supply a list of scenarios, you can just define the job and we don't to the foreach loop. The yaml comment is more intended for what I predict is the next stress test author copy/pasting from this one.